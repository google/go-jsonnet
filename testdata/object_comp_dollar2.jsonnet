{ [x]: { a: 1, b: if x == "a" then 42 else $.a.b + 1 } for x in ["a", "b", "c"] }
