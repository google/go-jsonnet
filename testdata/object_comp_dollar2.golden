{
   "a": {
      "a": 1,
      "b": 42
   },
   "b": {
      "a": 1,
      "b": 43
   },
   "c": {
      "a": 1,
      "b": 43
   }
}
